app/models/expenditure.rb

```diff
@@ -0,0 +1,15 @@
+class Expenditure < ApplicationRecord
+  belongs_to :user
+  belongs_to :group
+
+  validates :name, presence: true
+  validates :amount, presence: true
+
+  def total_expenditures(group_id)
+    total = 0
+    Expenditure.where(group_id:).each do |expenditure|
+      total += expenditure.amount
+    end
+    total
+  end
+end
```

Comment:
- [ ] Apparently, there is a small issue in this `expenditure.rb` file which is causing the server to not run please address that
![err](https://user-images.githubusercontent.com/44745855/159767173-62181624-7975-4d49-8871-9cf65ef5569f.png)

Reply:
I had `Expenditure.where(group_id: group_id)` but rubocop decided to get of one `group_id` 😆

Reply:
@ShoiraTa Instead of group_id you can just write id. It won't be a problem.

Reply:
yeah I did that and pushed 👍🏻